docs/kubernetes/user-guides/basic.md

```diff
@@ -85,3 +85,9 @@ $ kubectl get pods
 NAME                      READY     STATUS    RESTARTS   AGE
 web-server                1/1       Running   0          1m
 ```
+
+## StorageClass Fields
+
+The list of recognized StorageClass `parameters` is the same as the list of [CSI CreateVolume parameters](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver#createvolume-parameters).
+
+The list of recognized topology keys in `AllowedTopologies` is listed [here](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver#topology)
```

Comment:
can you clarify what "AllowedTopologies" is here, or link to some docs about what this could be for either here or in the linked spot